apps/coordinator/src/components/Wallet/ExtendedPublicKeyImporter.jsx

```diff
@@ -82,6 +82,7 @@ class ExtendedPublicKeyImporter extends React.Component {
             variant="standard"
             onChange={this.handleMethodChange}
           >
+            <MenuItem value={BITBOX}>BitBox</MenuItem>
```

Comment:
Noticed we will need this option in the wallet spend—specifically `apps/coordinator/src/components/ScriptExplorer/SignatureImporter.jsx` 

Suggesting:
```js
// Line 123
<MenuItem value={BITBOX}>BitBox</MenuItem>

// Line 158
if (method === TREZOR || method === LEDGER || method === BITBOX) {
  return (
    <DirectSignatureImporter
           ...
```

<img width="914" alt="Screenshot 2024-08-31 at 20 54 34" src="https://github.com/user-attachments/assets/8674ffa9-b7d7-4706-9cdc-268300a263c7">

Reply:
I made the change, but I couldn't quickly figure out how to find this in the UI. Could you you test it?

Reply:
Looks great—thanks! It's accessible under the spend tab after loading the wallet + building a transaction
<img width="1226" alt="Screenshot 2024-09-02 at 22 47 07" src="https://github.com/user-attachments/assets/83965b8e-e4d5-4d3d-98c7-65dc5fd91535">

Reply:
An additional consideration is we may want to hide BitBox in the various menus when using P2SH—attached a quick patch to assist with this
[hide_on_p2sh.patch](https://github.com/user-attachments/files/16933830/hide_on_p2sh.patch)

Reply:
Thanks, I applied the patch :+1: